Cython/Utility/ExtensionTypes.c

```diff
@@ -74,6 +74,41 @@ static int __Pyx_PyType_Ready(PyTypeObject *t) {
     return r;
 }
 
+/////////////// TRASHCAN.proto ///////////////
```

Comment:
> No need to scream. ;-) Just name it `PyTrashcan` or `PyTrashcanSupport`.

I prefer to use `TRASHCAN` in the name because the upstream macros are also called `Py_TRASHCAN_SAFE_BEGIN` and `Py_TRASHCAN_SAFE_END`. It will be more confusing if Cython uses a different made-up name.

Reply:
I was referring to the name of the utility code snippet, not the C macros (where upper case is normal). There is no precedent for an all-upper-case utility code name.